lib/dialects/mssql/data-types.js

```diff
@@ -135,9 +135,13 @@ module.exports = BaseTypes => {
   inherits(DATE, BaseTypes.DATE);
 
   DATE.prototype.toSql = function toSql() {
-    return 'DATETIMEOFFSET';
```

Comment:
?

Reply:
Sorry for the mixup and confusion: I started doing changes for a project of ours in the first place. The PR contained the initial changes we needed but we went further. The mistake was that I forgot that if I commit and push the new changes, this PR will also be updated.

Moved the new -and unrelated to this PR- changes to a different branch and reverted the changes you questioned.